server/api/jellyfin.ts

```diff
@@ -0,0 +1,246 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import axios, { AxiosInstance } from 'axios';
+import logger from '../logger';
+
+export interface JellyfinUserResponse {
+  Name: string;
+  ServerId: string;
+  ServerName: string;
+  Id: string;
+  PrimaryImageTag?: string;
```

Comment:
```suggestion
  name: string;
  serverId: string;
  serverName: string;
  id: string;
  primaryImageTag?: string;
```
Please modify the naming of the interface properties in this file to have a lowercase first letter, to follow the pattern elsewhere in the project.

Reply:
the properties provided by Jellyfin are capitalized, how should I go about doing this?

Reply:
Ah whoops, nevermind then.